### ISSUE TITLE
Fine-tuning the body during its life

### ISSUE THREAD

**Issue by denischergicalo** [google-research/self-organising-systems#8]
Hello! I admire your work and see great potential for research and future technologies in it.

In your paper "Transformers learn in-context by gradient descent" you show that transformers have a semblance of gradient descent in a forward pass.
As you can see, is it possible to represent some of the body's self-regulation processes as an autoregressive model, such as a transformer?

And if so, does the organism exhibit its adaptation similar to gradient descent by the error of simple autoregression?
